snuba/datasets/configuration/issues/storages/search_issues.yaml

```diff
@@ -124,7 +124,7 @@ query_processors:
 
 deletion_settings:
   is_enabled: 1
-  max_rows_to_delete: 100000
+  max_rows_to_delete: 2000000
```

Comment:
Is this change intentional? Are we saying that since rate limiter is enforced, we are increasing the default no. of rows to be deleted for search issues?

Reply:
this is intentional, it's to match the value passed in ops https://github.com/getsentry/ops/blob/27d30b56d219c1655d2fac9361f0d0a6eb4c708c/k8s/services/snuba/_values.yaml#L558